### PR TITLE
Make hostname configurable via sonar.perforce.clientImpersonatedHostname

### PR DESCRIPTION
**Pull request by mjdetullio**
Provides the equivalent mechanism for setting P4HOST environment
variable or using the -H global option.

... don't ask why I needed it... I'm pretty sure Perforce figured out it hadn't given me a big enough headache recently so it threw me this curveball

### CODE REVIEW

File: src/main/java/org/sonar/plugins/scm/perforce/PerforceConfiguration.java [hunk 1]
```diff
@@ -90,6 +91,15 @@ public PerforceConfiguration(Settings settings) {
         .subCategory(CATEGORY_PERFORCE)
         .index(3)
         .build(),
+      PropertyDefinition.builder(CLIENT_IMPERSONATED_HOST_PROP_KEY)
+        .name("Client impersonated hostname")
+        .description("Name of the host computer to impersonate, per the <a href=\"https://www.perforce.com/perforce/r15.1/manuals/cmdref/P4HOST.html\">Perforce documentation</a>")
```
Comment (henryju) [inline src/main/java/org/sonar/plugins/scm/perforce/PerforceConfiguration.java]: Do you confirm you have checked that HTML is allowed (and actually work) in property description?

**Comment by henryju** [comment]
Ticket created: https://jira.sonarsource.com/browse/SONARSCPER-16

File: src/main/java/org/sonar/plugins/scm/perforce/PerforceConfiguration.java (hunk 1 above)
Comment (mjdetullio) [inline src/main/java/org/sonar/plugins/scm/perforce/PerforceConfiguration.java]: Yes, descriptions accept HTML and I tested the link in the UI.

**Comment by henryju** [comment]
LGTM

**Pull request merged**